README.md

```diff
@@ -2,13 +2,26 @@
 
 Currently we support python3 only.
 
+### Dependencies maintaince
+
+Since elastic beanstalk use `requirements.txt` to install packages by default and there is no way to change it.
+We can't use [`constraints.txt` concept](https://stackoverflow.com/questions/34645821/pip-constraints-files) directly for clear dependency organization.
+
+If you want to add new package to this project, follow the steps below:
+
+0. activate the virtual env
+1. `pip install xxx`
+2. add new item for that package in `requirements/base.txt` or `requirements/dev.txt`
+
```

Comment:
Don't we need to adjust requirementx.txt afterward?

Reply:
no, we have a pre-commit hook to update requirements.txt each time.